internal/ldcache/ldcache.go

```diff
@@ -47,6 +47,11 @@ const (
 	flagArchX8664   = 0x0300
 	flagArchX32     = 0x0800
 	flagArchPpc64le = 0x0500
+
+	// flagArch_ARM_LIBHF is the flag value for 32-bit ARM libs using hard-float.
+	flagArch_ARM_LIBHF = 0x0900
+	// flagArch_AARCH64_LIB64 is the flag value for 64-bit ARM libs.
+	flagArch_AARCH64_LIB64 = 0x0a00
```

Comment:
Yup, this is how I fixed it, should you all carry on all the flags that `libnvidia-container` supports?

Reply:
Yes, we could do that. Although we don't actively test on some of the platforms called out there anymore.

I will audit the list once more.

Reply:
@arnaldo2792 I think for the time being I'm not going to update this list and only stick to ARM. If you have experience with other platforms that we should address, please let me know.